src/Libraries/OrchardCore.Commerce.AddressDataType/Abstractions/IAddressFormatter.cs

```diff
@@ -11,15 +8,5 @@ public interface IAddressFormatter
     /// <summary>
     /// Returns a multi-line formatted string representing the <paramref name="address"/>.
     /// </summary>
-    string Format(Address address) =>
-        address is null
-        ? "-"
-        : JoinNotNullAndNotWhiteSpace(
-            Environment.NewLine,
-            address.Department,
-            address.Company,
-            address.StreetAddress1,
-            address.StreetAddress2,
-            JoinNotNullAndNotWhiteSpace(separator: " ", address.City, address.Province, address.PostalCode),
-            address.Region).ToUpperInvariant();
+    string Format(Address address);
```

Comment:
Why did you remove the default implementation?

Reply:
Well this was hard to track :(
So basically this warning is the cause : 
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/92299130/27f13695-2b22-41d1-bed8-bc4069aa88a1)

The warning wants us to use concrete types. The default implementation was not in the concrete type but the interface. So I moved the default inmplementation to the inheritor class instead of the interface. I thought we don't do implementations in interfaces. Looks pretty wierd. I guess it had a reason though. So it was moved to the `DefaultAddressFormatter : IAddressFormatted` class.

Reply:
I looked into the git history of the file and looks like this was from before we took over. I don't think this was ever a good use of the default implementation feature, because this is the only method in the interface so the only reason to create a new class implementing it is to "override" this implementation anyway.
So it was a good thing you did here.